udata/core/owned.py

```diff
@@ -24,6 +24,19 @@ def owned_by(self, *owners):
             qs |= Q(owner=owner) | Q(organization=owner)
         return self(qs)
 
+    def visible_by_user(self, user: User, visible_query: Q):
+        """Return EVERYTHING visible to the user."""
```

Comment:
I think we may want to add an admin case.
```
if user.sysadmin:
    return self()
```

Reply:
Nice addition, indeed. I added it and the test that goes with it 👍

Reply:
Thank you for the fix when user isn't authenticated! :p